### PR TITLE
Dicom export rework

### PR DESCRIPTION
**Pull request by lassoan**
Move Slicer Data Bundle DICOM export to a plugin.
Also added a few other minor fixes and improvements (just running through this pull request to get these tested, too).

### CODE REVIEW

File: Docs/user_guide/modules/dicom.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
+# DICOM
+
+## Overview
+
+This module allows importing and exporting and network transder of DICOM data. Slicer provides support for the most commonly used subset of DICOM functionality, with the particular features driven by the needs of clinical research: **reading** and **writing** data sets from/to disk in DICOM format and network transfer - **querying**, **retrieving**, and **sending** and **receiving** data sets - using DIMSE and DICOMweb networking protocols.
```
Comment (jcfr) [inline Docs/user_guide/modules/dicom.md:5]: transder-> transfer

File: Docs/developer_guide/modules/dicom.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The existing two rules can be used as examples: the [default](https://github.com/commontk/CTK/blob/9c2af28e84da1abb986036317d75009d4c149923/Libs/DICOM/Core/ctkDICOMDisplayedFieldGeneratorDefaultRule.cpp) and the [RT](https://github.com/commontk/CTK/blob/9c2af28e84da1abb986036317d75009d4c149923/Libs/DICOM/Core/ctkDICOMDisplayedFieldGeneratorRadiotherapySeriesDescriptionRule.cpp) rules.
+
+## Plugin architecture
+
+From a developer perspective, the DICOM module exposes a plug-in architecture that allows acquisition-specific and modality-specific interpretation of DICOM data.
```
Comment (jcfr) [inline Docs/developer_guide/modules/dicom.md:23]: Link back to user plugin doc

File: Docs/user_guide/get_help.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Get Help
+
+[Contact the Slicer community or commercial partners](about.html#contact-us) if you have any questions, bug reports, or enhancement requests.
+
+## I need help in using Slicer
+
+- I have a question about how to use Slicer for a particular task or I'm not sure things are working right:
+  - Type your queston into Google. There is a good chance that your question has been asked and anwered before and all questions ever asked about Slicer are publicly available and indexed by Google.
```
Comment (jamesobutler) [inline Docs/user_guide/get_help.md:8]: Unclear if some of the first DOC commits are intended for this PR as they were originally included in https://github.com/Slicer/Slicer/pull/5268

Still the issue of “queston” here as mentioned in a review on that other PR.

**Review by pieper — DISMISSED** [review]
Looks good to me.  Looking forward to seeing a demo; maybe during the developer call on Tuesday.

File: Docs/user_guide/get_help.md (hunk 3 above)
Comment (lassoan) [reply Docs/user_guide/get_help.md:8]: Good catch! It seems that I accidentally included that same commit in this PR, too (was in the same integration branch). I'll fix your comments before merging. Thank you for the review!

**Comment by lassoan** [comment]
Thank you for all the reviews. Comments are fixed and the PR is integrated.

**Pull request merged**